seata/go.mod

```diff
@@ -1,8 +1,8 @@
 module github.com/apache/dubbo-go-samples/seata
 
 require (
-	github.com/apache/dubbo-go v1.4.1
-	github.com/apache/dubbo-go-hessian2 v1.6.0
+	github.com/apache/dubbo-go v1.5.2-0.20201205063214-68ff8be79d6d
+	github.com/apache/dubbo-go-hessian2 v1.7.0
```

Comment:
Could you move this require to [root/go.mod](https://github.com/apache/dubbo-go-samples/blob/master/go.mod) ?